samples/index.html

```diff
@@ -64,5 +73,16 @@
 
   <footer>
     <script src="./assets/js/AnimationSystem.js"></script>
+    <script>
+
+      let model = document.querySelector('#logo-model')
+
+      function spin() {
+        let animationComp = model.components.get('animation')
+        model.components.set('animation', {maxspeed: animationComp.maxspeed * 10000, acceleration: animationComp.acceleration * 10000})
+      }
+
```

Comment:
lines 75-84 include two different ways to do animation - are we keeping both?

Reply:
This function doesn't actually animate anything, it changes the animation component values. I'll add a comment.